src/lib/PhpCsFixer/Config.php

```diff
@@ -10,182 +10,17 @@
 
 use PhpCsFixer\Config as ConfigBase;
 
+/**
+ * @deprecated Use RuleSetInterface or InternalConfigFactory instead
```

Comment:
Please follow the standards from https://ibexa.atlassian.net/wiki/spaces/ENG/pages/5799937/PHP+Conventions+Deprecation+standards (in which version class has been deprecated, when it will be removed etc).

Reply:
Fixed in 6d135d07.